src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -189,6 +189,11 @@
     "is_application": PREFIX + 269,
     "platform": PREFIX + 270,
     "os.version": PREFIX + 271,
+    "mri": PREFIX + 272,
+    "mri.type": PREFIX + 273,
+    "mri.namespace": PREFIX + 274,
+    "outcome": PREFIX + 275,
+    "reason": PREFIX + 276,
```

Comment:
`reason` is already used, but only in `custom` metrics (which is still in alpha) AND for a single org in the `transactions` usecase with seemingly low volume